sql/derived_tables/loans_items.sql

```diff
@@ -29,7 +29,7 @@ SELECT
     cl.loan_date,
     cl.due_date AS loan_due_date,
     cl.return_date AS loan_return_date,
-    json_extract_path_text(cl.data, 'systemReturnDate') AS system_return_date,
+    json_extract_path_text(cl.data, 'systemReturnDate')::timestamptz at time zone 'UTC' AS system_return_date,
```

Comment:
The addition of `at time zone 'UTC'` converts the `timestamptz` to `timestamp without time zone`.  It would probably be better to leave it as `timestamptz` in a derived table.